_rules/heading-descriptive-b49b2e.md

```diff
@@ -136,12 +136,12 @@ Heading marked up with `h1` element that is a single character that describes th
 
 #### Passed Example 6
 
-Heading marked up with `role="heading"` that describes the topic or purpose of the following [palpable content][]. The heading is positioned off screen and is [included in the accessibility tree][].
+Heading marked up with `role="heading"` that describes the topic or purpose of the following [palpable content][]. The heading is positioned off screen but is [included in the accessibility tree][].
 
 ```html
 <html lang="en">
 	<span role="heading" aria-level="1" style="position: absolute; top: -9999px; left: -9999px;">Opening Hours</span>
-	<p style="position: absolute; top: -9999px; left: -9999px;">
+	<p>
```

Comment:
I'm not fully sure either why this (and FE3) were having the paragraph off-screen. I guess the idea was that both the heading and its "section" were `sr-only` 🤔 Might be a bit borderline for 2.4.10 otherwise (since the visual content does not have headings to organise content, it's OK for a 2 lines toy examples but would be bad if there were 5 or 6 of these in a page).

OTOH, if that's the intention, it might be cleaner and clearer to wrap both the heading and the paragraph into the sami `div` and move that `div` off screen instead…

Anyway, no real opposition to this change.